opencti-platform/opencti-front/src/private/components/assets/devices/DeviceEditionContainer.js

```diff
@@ -122,23 +123,37 @@ class DeviceEditionContainer extends Component {
   }
 
   onSubmit(values, { setSubmitting, resetForm }) {
+    const filteredValue = {};
+    const { totalInitial } = this.state;
     const adaptedValues = R.evolve(
       {
-        release_date: () => parse(values.release_date).format(),
+        release_date: () => values.release_date === null ? null : parse(values.release_date).format(),
       },
       values,
     );
+    Object.keys(totalInitial).forEach((key, j) => {
+      if (Array.isArray(adaptedValues[key])) {
+        if (adaptedValues[key].some((value, i) => value !== totalInitial[key][i])) {
+          filteredValue[key] = adaptedValues[key];
+        }
+      }
+      if (!Array.isArray(adaptedValues[key]) && totalInitial[key] !== adaptedValues[key]) {
+        filteredValue[key] = adaptedValues[key];
+      }
+    });
     const finalValues = R.pipe(
+      R.dissoc('id'),
       R.dissoc('locations'),
       R.dissoc('protocols'),
       R.dissoc('port_number'),
       R.dissoc('responsible_parties'),
       R.toPairs,
       R.map((n) => ({
         'key': n[0],
-        'value': adaptFieldValue(n[1]),
+        'value': Array.isArray(adaptFieldValue(n[1])) ? adaptFieldValue(n[1]) : [adaptFieldValue(n[1])],
       })),
-    )(adaptedValues);
+    )(filteredValue);
+    console.log('finalValues', finalValues);
```

Comment:
Please do not include `console.log` in your PR's.

Reply:
@AMAN-SHARMA-82591 @sachin755  Did you see the request from Dallas regarding the console.log, could we remove these. Thanks guys.